otx/algorithms/classification/utils/data.py

```diff
@@ -60,6 +61,8 @@ class ClassificationDatasetAdapter(DatasetEntity):
             "val_data_root": OptionalDirectoryPathCheck,
             "test_ann_file": OptionalDirectoryPathCheck,
             "test_data_root": OptionalDirectoryPathCheck,
+            "ul_data_root": OptionalDirectoryPathCheck,
```

Comment:
I think it's better to use unlabeled_data_loot than jsut ul_data_root to increase readability.
how do you think about?

Reply:
Fixed. Please check.